updated integration content (#601)
helium/docs: docs/use-the-network/console/integrations/mydevices-cayenne.mdx

The next step is to name the integration

:::info 
**Downlink Not Yet Supported**  
The MyDevices integration currently only supports uplinks. Downlinks are in the
process of being implemented and will be available soon. 
:::

## Connecting Integrations to Devices
Devices or groups of devices (via labels) can be directly connected to integrations on the Flows Worskpace. Labels are identifiers used to group devices for easy management. To connect one or more devices to one or more integrations, simply connect the Device and Integration nodes on the Flows Workspace.

:::info
Quick video tutorial connecting devices to an integration [here](https://www.youtube.com/watch?v=lnERw1f7TGE).
:::

Node elements (devices, labels, integrations) need to be created before they're available on the Flows Workspace. More information about Flows [here](https://docs.helium.com/use-the-network/console/flows/).

### Adding Your Device on myDevices Cayenne

Before adding your device to Cayenne, be sure you've successfully added it to

> the Helium network as well as connected it the myDevices Cayenne integration, if you haven't done so yet, then see how to

Reviewer: ```suggestion
the Helium network as well as connected it the myDevices Cayenne integration. If you haven't done so yet, start
```
